game.project

```diff
@@ -15,3 +15,5 @@ _dependencies = https://github.com/GameAnalytics/defold-openssl/archive/1.0.0.zi
 [library]
 include_dirs = websocket
 
+[websocket]
+debug = 2
```

Comment:
Should it be checked in?
Might be interpreted as "needed" by users?

Reply:
Good point. Removed it.